spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/manager/BackingAppManagementService.java

```diff
@@ -82,8 +114,24 @@ public BackingAppManagementService(ManagementClient managementClient, AppDeploye
 			.then();
 	}
 
+	/**
+	 * Helper method that fetches service name and plan name from Cloud Foundry Service Instances API (CF API) and
+	 * invokes {@code restart(serviceInstanceId, serviceName, planName)}.
```

Comment:
By referencing e.g. `restart(serviceInstanceId, serviceName, planName)` in this Javadoc it makes me think we should Javadoc those methods too. Just something simple like 'Restarts the backing applications for the service instance with the given id'.